README.md

```diff
@@ -17,3 +17,34 @@ Fork this repository, and submit your pull request providing documentation to th
 
 ### Product Specification
 * See wireframe.png
+
+############################
+
+* I was really unsure of how in detail I was supposed to go and I only had a a few of hours to work on this
```

Comment:
The limit date was today, Monday 10AM, as specified in the email.

Reply:
True, but I guess I was not sure if I was to spend time doing input validation, testing, extensive documentation..... But I guess I should know the more I do the better.